save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/entities/vulnerability/VulnerabilityProjectDto.kt

```diff
@@ -3,9 +3,9 @@ package com.saveourtool.save.entities.vulnerability
 import kotlinx.serialization.Serializable
 
 /**
- * @property name
- * @property url
- * @property isOpen
+ * @property name of project
+ * @property url of project
+ * @property isOpen source project
```

Comment:
I still don't get which project is used here. Suggestion to mention that it is not `Project` entity. Moreover, I would like you to write better kdocs as it is really not clear what and why is it. 

> isOpen source project

I cannot get what does it mean.